### ISSUE TITLE
Bug: Toot Share Button Doesn't Have Copy Link

### ISSUE THREAD

**Issue by jonathankoren** [Dimillian/IceCubesApp#1538]
## Environment:

 - OS:  17.0
 - IceCubesApp version: 1.7.1
 - 
## Description

The per toot share button doesn't provide a way to simply copy the link. You can share it directly to some app, or you can copy the text. You can't get a link. This is inconsistent with every other share button on iOS.

Also, no one wants to the copy the text. There's a million ways to do that in this app. But getting the link to a toot, is not obvious at all.

## Related Issues

None

**Comment by AxelTerizaki** [comment]
I often find myself having to delete the text manually when wanting to post a link to a toot somewhere, so this change would be very welcome if it gets into the app :)

**Comment by Klikini** [comment]
Right-clicking the toot itself has a "Copy link" option, but it does nothing, so I currently do this by clicking "View in browser" instead and then copying the URL from the address bar.

![screenshot of the toot context menu with "Copy link" highlighted](https://github.com/user-attachments/assets/ca2d31a7-877f-4d3d-b1e9-d0338ede1f7f)
